README.md

```diff
@@ -61,7 +69,7 @@ and
 }
 ```
 
-## Azure Storage
+### Azure Storage
```

Comment:
Woudl explictly say "Azure Blob Storage" (there are 3/4 types of Azure Storage, i.e. Table, File, etc)